modules/exploits/multi/http/oscommerce_installer_unauth_code_exec.rb

```diff
@@ -91,8 +91,11 @@ def exploit
     }
 
     res = send_request_cgi({
-      'uri'       => normalize_uri(datastore['URI'], 'install.php?step=4'),
+      'uri'       => normalize_uri(datastore['URI'], 'install.php'),
       'method'    => 'POST',
+      'vars_get'  => {
+        'step' => '4'
```

Comment:
The number doesn't have to be a string, since it's coerced into one, but this is good.